### DISCUSSION TITLE
问下这种情况是小狼毫还是输入方案的问题

### DISCUSSION THREAD

**Discussion by 971230** [rime/home#1505] [General]
就是我们电脑是长时间没有重启了，昨天处于需要重启了电脑，但是之后小狼毫输入「wlw」，之前还可以出现「物联网」，现在只有这样了，没有任何的候选项
![image](https://github.com/rime/home/assets/37500200/3291ccdd-8755-40a6-961d-6770f99c9275)
输入「wulianw」这样，也是如图，问下有没有懂得各位，这个是输入方案问题还是小狼毫的问题，我开了「雾凇」和「小鹤双拼」两种方案，启用了雾凇
![image](https://github.com/rime/home/assets/37500200/6a9d05ff-d422-47cf-ad1c-3837f3dbc38d)

**Comment by yanhuacuo** [comment]
算法服务退出了，托盘，重启算法服务即可。

**Reply by 971230** [reply]
谢谢兄弟
